src/coreclr/jit/hwintrinsicxarch.cpp

```diff
@@ -2403,8 +2403,13 @@ GenTree* Compiler::impSpecialIntrinsic(NamedIntrinsic        intrinsic,
 
             if (!varTypeIsFloating(simdBaseType))
             {
-                // We can't trivially handle division for integral types using SIMD
-                break;
+                // Check to see if it is possible to emulate the integer division
+                if (!(simdBaseType == TYP_INT &&
+                      ((simdSize == 16 && compOpportunisticallyDependsOn(InstructionSet_AVX)) ||
```

Comment:
nit: Ideally we'd support doing this on pre-AVX and pre-AVX512 hardware by doing 2x 128-bit or 2x 256-bit operations rather than requiring a full widen.

This is beneficial for AOT and ReadyToRun scenarios where lower baselines are typically targeted by default.